docs/UML/Commands/commandExit.puml

```diff
@@ -0,0 +1,18 @@
+@startuml
+!theme plain
+title Command Exit Sequence Diagram
+
+actor User
+participant Ui
+participant CommandExit
+participant Storage
+
+User -> Ui: Decides to exit the application
+Ui -> CommandExit: Executes "bye" command
+CommandExit -> Ui: Signals the program to exit
+Ui -> Storage: Saves any unsaved data
+Storage --> Ui: Data saved successfully
+Ui -> Ui: Displays exit message
```

Comment:
You should add an additional activation bar when calling a method inside the same class (see screenshot below):
<img width="181" alt="image" src="https://github.com/nus-cs2113-AY2324S1/tp/assets/62311026/a4455903-b206-4dc2-884f-d08baf312a55">